benchmarks/gravity_mantle/mantle_gravity.prm

```diff
@@ -0,0 +1,87 @@
+# A simple setup for testing the gravity postprocessor using the
+# sample scheme 'map'.
+# See the corresponding section in the manual for more information.
+
+# General parameters
+set Dimension                              = 3
+set End time                               = 0
+set Output directory                       = output_shellgravity_S40RTS_new
+set Nonlinear solver scheme                = no Advection, no Stokes
+
+# Model geometry
+subsection Geometry model
+  set Model name = spherical shell
+  subsection Spherical shell
+    set Inner radius  = 3480e3
+    set Outer radius  = 6251e3
+  end
+end
+
+# Model boundary velocity
+subsection Boundary velocity model
+  set Zero velocity boundary indicators = top, bottom
```

Comment:
Since you dont solve the stokes equation, remove the whole section. Or does the code complain in that case?